tests/system-tests/rdscore/internal/rdscorecommon/log-forwarding.go

```diff
@@ -0,0 +1,265 @@
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("operator deployment %s failure in the namespace %s; %v",
+			rdscoreparams.CLOName, rdscoreparams.CLONamespace, err))
+
+	By("Retrieve kafka server URL")
+	glog.V(rdscoreparams.RDSCoreLogLevel).Infof("Retrieve Kafka server URL from the ClusterLogForwarder")
+
+	clusterLogForwarder, err := clusterlogging.PullClusterLogForwarder(
+		APIClient, rdscoreparams.CLOInstanceName, rdscoreparams.CLONamespace)
+	Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(
+		"Failed to retrieve ClusterLogForwarder %s from the namespace %q; %v",
+		rdscoreparams.CLOInstanceName, rdscoreparams.CLONamespace, err))
+
+	clfOutput := clusterLogForwarder.Object.Spec.Outputs
+	Expect(len(clfOutput)).ToNot(Equal(0), fmt.Sprintf(
+		"No collector defined in the ClusterLogForwarder %s from the namespace %q",
+		rdscoreparams.CLOInstanceName, rdscoreparams.CLONamespace))
+
+	var kafkaURL, kafkaUser string
+
+	for _, collector := range clfOutput {
+		if collector.Type == "kafka" {
+			clfKafkaURL := collector.Kafka.URL
+
+			glog.V(100).Infof("collector.URL: %s", clfKafkaURL)
+
+			kafkaURL = strings.Split(clfKafkaURL, "/")[2]
+			kafkaUser = strings.Split(clfKafkaURL, "/")[3]
+
+			break
+		}
+	}
+
+	kafkaLogsLabel := RDSCoreConfig.KafkaLogsLabel
+
+	if kafkaLogsLabel == "" {
+		By("Getting cluster domain")
+
+		clusterDNS, err := dns.Pull(APIClient)
+		Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(
+			"Failed to retrieve clusterDNS object cluster from the namespace default; %v", err))
+
+		clusterDomain := clusterDNS.Object.Spec.BaseDomain
+		Expect(clusterDomain).ToNot(Equal(""), "cluster domain is empty")
+
+		glog.V(100).Infof("DEBUG: clusterDomain: %s", clusterDomain)
+
+		kafkaLogsLabel = clusterDomain
+	}
+
+	By("Build query request command")
+
+	cmdToRun := []string{"/bin/sh", "-c", fmt.Sprintf("kcat -b %s -C -t %s -C -q -o end -c %d | grep %s",
+		kafkaURL, kafkaUser, logMessageCnt, kafkaLogsLabel)}
+
+	By("Retrieve kcat pod object")
+
+	kcatPodObj, err := getPodObjectByNamePattern(APIClient, kcatDeploymentName, kcatDeploymentNamespace)
+	Expect(err).ToNot(HaveOccurred(),
+		fmt.Sprintf("Failed to retrieve %s pod object from namespace %s: %v",
+			kcatDeploymentName, kcatDeploymentNamespace, err))
+
+	By("Retrieve logs forwarded to the kafka")
+
+	glog.V(rdscoreparams.RDSCoreLogLevel).Infof("Execute command: %q", cmdToRun)
+
+	var result string
+
+	Eventually(func() bool {
+		output, err := kcatPodObj.ExecCommand(cmdToRun, kcatPodObj.Object.Spec.Containers[0].Name)
+
+		if err != nil {
+			glog.V(rdscoreparams.RDSCoreLogLevel).Infof(
+				"Error running command from within a pod %q in namespace %q: %v",
+				kcatPodObj.Definition.Name, kcatPodObj.Definition.Namespace, err)
+
+			return false
+		}
+
+		glog.V(rdscoreparams.RDSCoreLogLevel).Infof(
+			"Successfully executed command from within a pod %q in namespace %q",
+			kcatPodObj.Definition.Name, kcatPodObj.Definition.Namespace)
+
+		result = output.String()
```

Comment:
I suggest adding extra check to ensure that output is not empty and return _false_ to retry command, wdyt?
```suggestion
		result = output.String()
		if result == "" {
		    glog.Info....
		    return false
		}
```